http-clients/src/test/java/com/palantir/remoting/http/serviceconfig/ExampleApp.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting.http.serviceconfig;
+
+import com.palantir.config.service.ServiceConfiguration;
+import com.palantir.config.service.ServiceDiscoveryConfiguration;
+import io.dropwizard.Application;
+import io.dropwizard.setup.Environment;
+
+/**
+ * Used to test {@link ServiceConfiguration} and {@link ServiceDiscoveryConfiguration}.
+ */
+public final class ExampleApp extends Application<ExampleAppConfiguration> {
```

Comment:
the test code is pretty heavy here. can we re-use some code from other projects by making a shared test-only project?